go/src/koding/klientctl/endpoint/machine/mount.go

```diff
@@ -201,25 +206,80 @@ func (c *Client) Umount(options *UmountOptions) (err error) {
 		return errors.New("invalid nil options")
 	}
 
-	// TODO: ask user to confirm unmounting.
-	fmt.Fprintf(os.Stdout, "Unmounting %s...\n", options.Identifier)
+	// Get all mounts if all option is set.
+	identifiers := options.Identifiers
+	if options.All {
+		mids, err := c.allMounts()
+		if err != nil {
+			return err
+		}
+		identifiers = mids
+	}
 
-	// Remove mount.
-	umountReq := &machinegroup.UmountRequest{
-		Identifier: options.Identifier,
+	// If there are no mounts we return nil.
+	if len(identifiers) == 0 {
+		fmt.Fprintf(os.Stdout, "There is nothing to unmount\n")
+		return nil
```

Comment:
I think it should return non-nil error when len(options.Identifiers) == 1, just to be backward-compatible with previous behaviour.